pysquared/protos/radio.py

```diff
@@ -45,3 +46,10 @@ def receive(self, timeout: Optional[int] = None) -> Optional[bytes]:
         :rtype: Optional[bytes]
         """
         ...
+
+    def modify_config(self, radio_config: RadioConfig) -> None:
```

Comment:
Taking in `RadioConfig` is a little concerning.

I'm concerned, in general, about config in pysquared right now. It has a lot of embedded business logic that probably shouldn't be there. For example, changing node and destination is only valid for the RFM9x radio library when running in LoRa mode. We are supporting other radios and those all have to share the same config.

I think we should probably just go with the config management that we have today for 2.x and try to address these concerns in a future 3.x release. Michael suggested having a config for each manager? Maybe that would be a better way to organize?

Regardless, I think taking in `RadioConfig` is OK for this release.